packages/q/qt5core/xmake.lua

```diff
@@ -35,21 +34,22 @@ package("qt5core")
                 libname = libname .. "_x86_64"
             elseif package:is_arch("arm64", "arm64-v8a") then
                 libname = libname .. "_arm64-v8a"
-            elseif package:is_arch("armv7", "armv7-v7a") then
+            elseif package:is_arch("armv7", "armeabi-v7a", "armeabi", "armv7-a", "armv5te") then
                 libname = libname .. "_armeabi-v7a"
             elseif package:is_arch("x86") then
                 libname = libname .. "_x86"
             end
 
-            links = {libname, "z"}
+            syslinks = "z"
         end
 
         return {
             qtdir = qt,
             version = qt.version,
             includedirs = {qt.includedir, path.join(qt.includedir, "QtCore")},
-            links = links or table.wrap(libname),
-            linkdirs = table.wrap(qt.libdir)
+            links = table.wrap(libname),
+            linkdirs = table.wrap(qt.libdir),
+            syslinks = syslinks and table.wrap(syslinks) or nil
```

Comment:
we need not wrap them, xmake support string and table.

Reply:
for links and linkdirs too?

Reply:
right, support `links = "z"` and `links = {"z", "xx"}`